README.md

```diff
@@ -87,6 +87,7 @@ metadata:
   namespace: kube-system
 data:
   DAEMON_SM_PLUGIN: "ufm" # Name of the subnet manager plugin
+  DAEMON_SM_PLUGIN_PATH: "/plugins" # Path to SM plugins folder
```

Comment:
where do you use it? I still see envDefault:"/plugins"

Reply:
This provides means to override the default plugins path.
its shown explicitly in the README and points to the plugins location inside the container image.
as seen [here](https://github.com/Mellanox/ib-kubernetes/blob/c82060e00b9b2e405ba1fc4a6886c59a5789372d/Dockerfile#L15).

This makes life easier when doing development, where i can just point this to my plugin build dir location.
(when not building image or deploying through k8s but invoking binary directly instead)

Reply:
so why not move "/plugins" to constant as you used it in envDefault as well?

Reply:
This is a README , what do you mean by constant ?
in the code its listed once.

Reply:
I am talking about pkg/config/config.go  line 19

Reply:
correction to my above response, i dont think its possible to use variables as input to envDefault
as its a string annotation that `caarlos0/env/v6` use

https://github.com/caarlos0/env#supported-types-and-defaults

Reply:
you are right